modules/references/client/components/read-references/Reference.js

```diff
@@ -65,6 +65,7 @@ export default function Reference({ reference }) {
               {t('Member since {{memberSince}}', {
                 memberSince: new Date(userFrom.created).getFullYear(),
               })}
+              {userFrom.gender && `, ${getGender(userFrom.gender)}`}
```

Comment:
"Other" gender label will probably read a bit funny and be confusing here, but ok to figure it out some time later.

Reply:
I think it will look weird on the profile as well, maybe we can add that to the string of other as a work around until we find something better? gender: Other?

Reply:
Yes, agreed. How about just "other gender"?

There might be some translation-difficulties with partial sentences with commas where translators don't get the whole sentence to see. Can that be improved somehow too?

Reply:
other gender is good, I'll push that until we find a better solution.

Reply:
Hey how about having a gender icon/symbol and then it can be defined in a pop up bubble after hovering over the symbol?

Reply:
Thanks for the idea!

The trouble with hover is that it doesn't work on mobile and is really basically hiding info until folks realize to hover, so it's best to not rely on it.

Feels like it can be just fine solved with text so let's not complicate it? :-)

Reply:
how about a symbol and text, without hovering, then it'll be clear it's about gender?

Reply:
Agree, Chagai was mentioning something about hovering, but definitely don't want to complicate anything further and yes, it won't work with mobile. As a solution, a text symbol could be added such as this, which is what we were suggesting really. Check the ones here: https://gender.wikia.org/wiki/Gender_Symbol